lib/DxilDia/DxcPixLiveVariables.cpp

```diff
@@ -29,6 +29,35 @@
 
 #include <unordered_map>
 
+namespace std {
+template <> struct hash<std::pair<llvm::DIScope *, llvm::DIScope *>> {
```

Comment:
Since we need to define a custom hashing function anyway, can we use a struct instead of a `std::pair`? It'll be easier to read at a glance what the two scopes are.

Reply:
That makes sense...